src/libsync/caseclashconflictsolver.cpp

```diff
@@ -75,7 +75,7 @@ void CaseClashConflictSolver::onRemoteDestinationFileAlreadyExists()
 {
     _allowedToRename = false;
     emit allowedToRenameChanged();
-    _errorString = tr("Cannot rename file because a file with the same name does already exist on the server. Please pick another name.");
+    _errorString = tr("Cannot rename file because a file with the same name does already exists on the server. Please pick another name.");
```

Comment:
I object, (s) is not needed with `does`. Though, Grammarly suggests to just have it like `Cannot rename the file because a file with the same name already exists on the server. Please pick another name` without `does`.